Documentation/network/clustermesh/mcsapi.rst

```diff
@@ -0,0 +1,165 @@
+.. _gs_clustermesh_mcsapi:
+
+*********************************
+Multi-Cluster Services API (Beta)
+*********************************
+
+.. include:: ../../beta.rst
+
+This tutorial will guide you to through the support of `Multi-Cluster Services API (MCS-API) <MCS_API_KEP>`_ in Cilium.
+
+.. _MCS_API_KEP: https://github.com/kubernetes/enhancements/blob/master/keps/sig-multicluster/1645-multi-cluster-services-api/README.md
+
+Prerequisites
+#############
+
+You need to have a functioning Cluster Mesh setup, please follow the guide
+:ref:`gs_clustermesh` to set it up.
+
+You first need to install the required MCS-API CRDs:
+
+   .. code-block:: shell-session
+
+      kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/mcs-api/refs/heads/4690877cc89bcfd9302d516ae43515f446d0aecf/config/crd/multicluster.x-k8s.io_serviceexports.yaml
+      kubectl apply -f https://raw.githubusercontent.com/kubernetes-sigs/mcs-api/4690877cc89bcfd9302d516ae43515f446d0aecf/config/crd/multicluster.x-k8s.io_serviceimports.yaml
+
+
+To install Cilium with MCS-API support, run:
+
+   .. code-block:: shell-session
```

Comment:
```suggestion
   .. parsed-literal::
```

The `|CHART_RELEASE|` substitution does not work inside a code-block. Same for the one below.